phys/module_ra_rrtmg_sw.F

```diff
@@ -10849,9 +10849,11 @@ SUBROUTINE RRTMG_SWRAD(                                        &
 
            if(iceflgsw.eq.5)then
               do k = kts, kte
-                 snow_mass_factor = 1.0                 
+                 snow_mass_factor = 0.99        ! Assume 1% of snow overlaps the cloud ice category
+                 gicewp = gicewp + (qs1d(k)*(1.0-snow_mass_factor) * pdel(ncol,k)*100.0 / gravmks * 1000.0)
                  if (resnow1d(ncol,k) .gt. 130.)then 
-                     snow_mass_factor = (130.0/resnow1d(ncol,k))*(130.0/resnow1d(ncol,k))
+                     snow_mass_factor = MIN(snow_mass_factor,                       &
+     &                         (130.0/resnow1d(ncol,k))*(130.0/resnow1d(ncol,k)))
```

Comment:
@gthompsnWRF @weiwangncar @dudhia 
Greg,
With a cursory glance, it looks like the identical mods could go into RRTMG_SW fast.